lib/challenge.js

```diff
@@ -48,7 +60,12 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
+  pet: "Neptune",
+  species: "feline",
+  breed: "domestic shorthair",
+  noise: "meow"
+  }
```

Comment:
Yes, but you're missing the `makeNoise` method. It might look something like this:
```javascript
var some_pet = {
...
  noise : "meow",
  makeNoise : function(){
    console.log(this.noise);
  }
...
}
```